skeleton/Gemfile

```diff
@@ -7,6 +7,10 @@ gem_sources.each { |gem_source| source gem_source }
 group :test do
   gem 'rake'
   gem 'puppet', ENV.fetch('PUPPET_VERSION', '~>4.8.0') # Default to SIMP 6
+
+  # Fix this when simp-rake-helpers is fully updated
+  gem 'facter', '~> 2.4.0'
```

Comment:
@nick-markowski Upstream is still broken, so we need to leave this here for now.